recipe/meta.yaml

```diff
@@ -35,6 +37,8 @@ outputs:
     version: {{ ucx_version }}
     build:
       number: {{ number }}
+      run_exports:
+        - {{ pin_subpackage("ucx", max_pin="x.x") }}
```

Comment:
@pentschev, does this pinning seem reasonable to you? Basically this pins the major and minor version, but let's the patch version float

Reply:
Sorry for the naive question, but what is this pinning exactly responsible for? If the package is building `1.12.1`, how would not including the same patch version here affect things elsewhere?

Reply:
It affects other packages that build against `ucx` and depend on it at runtime. So if a package like `openmpi` builds with `ucx` version `1.12.1`, they will depend on `ucx >=1.12.1,<1.13.0` at runtime.

Reply:
Got it. I think that should be fine then, as API/ABI should be consistent among patch releases.